enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -43,6 +58,9 @@ const (
 	podmanDefaultNetworkIPRange = "10.88.0.0/16"
 	podmanDefaultNetworkGateway = "10.88.0.1"
 	podmanDefaultNetworkBridge  = "cni-podman0"
+
+	pollCIDTimeout    = 15 * time.Second
+	statsPollInterval = 50 * time.Millisecond
```

Comment:
please add comments about what this control

Reply:
Done